src/main/java/co/rsk/federate/signing/hsm/BlockStoreException.java

```diff
@@ -0,0 +1,12 @@
+package co.rsk.federate.signing.hsm;
```

Comment:
I'm not sure about this. The BlockStore is part of RSK, and we are adding here a BlockStoreException inside the HSM signing package. Doesn't seem related.

Perhaps we could use the already existing HSMBlockchainBookkeepingRelatedException, that one seems to make sense for this case. What do you think?

Reply:
Yes I Think Using The Existing HSMBlockchainBookkeepingRelatedException Is Better. Updated